operator/v1/types_ingress.go

```diff
@@ -0,0 +1,119 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ClusterIngress describes a discrete, managed ingress component for the
+// cluster. This includes a router, and may also include a load balancer and
+// external wildcard DNS records for that load balancer.
+type ClusterIngress struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   ClusterIngressSpec   `json:"spec,omitempty"`
+	Status ClusterIngressStatus `json:"status,omitempty"`
+}
+
+// ClusterIngressSpec is the specification of the desired behavior of the
+// ClusterIngress.
+type ClusterIngressSpec struct {
+	// IngressDomain is a DNS name suffix serviced by the ClusterIngress. If
```

Comment:
All of your godoc for public API fields should use `ingressDomain` camelCase for field names.

Reply:
Fixed